src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -5688,9 +5687,11 @@ namespace HVACVariableRefrigerantFlow {
             }
             // test that the system is active if constant fan logic enables system when thermostat control logic did not
             if (!CoolingLoad(VRFCond) && !HeatingLoad(VRFCond)) {
-                if (TerminalUnitList(TUListIndex).HRCoolRequest(IndexToTUInTUList)) {
+                if (TerminalUnitList(TUListIndex).HRCoolRequest(IndexToTUInTUList) && OutsideDryBulbTemp >= VRF(VRFCond).MinOATCooling &&
+                    OutsideDryBulbTemp <= VRF(VRFCond).MaxOATCooling) {
                     CoolingLoad(VRFCond) = true;
-                } else if (TerminalUnitList(TUListIndex).HRHeatRequest(IndexToTUInTUList)) {
+                } else if (TerminalUnitList(TUListIndex).HRHeatRequest(IndexToTUInTUList) && OutsideDryBulbTemp >= VRF(VRFCond).MinOATHeating &&
+                    OutsideDryBulbTemp <= VRF(VRFCond).MaxOATHeating) {
```

Comment:
This makes sense to me.  I'm surprised it's been lingering this long but I guess it's because users just didn't look close enough at the fan operation during heat recovery mode?

Reply:
That's also what I suspect, users looking at annual energy without inspection of detailed results. This means we really do need to be careful with flow of logic in all models since users won't typically catch these types of things.